lib/src/user/user_settings.dart

```diff
@@ -57,6 +57,10 @@ import 'user_bloc.dart';
 import 'user_event_state.dart';
 
 class UserSettings extends StatefulWidget {
+  static get viewTitle => L10n.get(L.profileEdit);
+
+  UserSettings();
```

Comment:
Shouldn't be required, right? If not please remove all empty constructors.

Reply:
True. Will change it 👍